homeassistant/components/vicare/strings.json

```diff
@@ -162,100 +162,112 @@
         "name": "DHW storage bottom temperature"
       },
       "hotwater_gas_consumption_today": {
-        "name": "DHW gas consumption today"
+        "name": "Gas consumption (DHW) today"
       },
       "hotwater_gas_consumption_heating_this_week": {
-        "name": "DHW gas consumption this week"
+        "name": "Gas consumption (DHW) this week"
       },
       "hotwater_gas_consumption_heating_this_month": {
-        "name": "DHW gas consumption this month"
+        "name": "Gas consumption (DHW) this month"
       },
       "hotwater_gas_consumption_heating_this_year": {
-        "name": "DHW gas consumption this year"
+        "name": "Gas consumption (DHW) this year"
       },
       "gas_consumption_heating_today": {
-        "name": "Heating gas consumption today"
+        "name": "Gas consumption (heating) today"
       },
       "gas_consumption_heating_this_week": {
-        "name": "Heating gas consumption this week"
+        "name": "Gas consumption (heating) this week"
       },
       "gas_consumption_heating_this_month": {
-        "name": "Heating gas consumption this month"
+        "name": "Gas consumption (heating) this month"
       },
       "gas_consumption_heating_this_year": {
-        "name": "Heating gas consumption this year"
+        "name": "Gas consumption (heating) this year"
       },
       "gas_consumption_fuelcell_today": {
-        "name": "Fuel cell gas consumption today"
+        "name": "Gas consumption (power) today"
```

Comment:
Should this be power or energy?

Reply:
This does not need to be changed at all. It's correct as it is.
All measured consumption is _energy_, but for gas this is shown in _cubic meters_.

But as this is part of the entity's value there is also no need to include the units in the names.
Here the example for my heating gas consumption today:

![image](https://github.com/user-attachments/assets/58336d81-5692-429f-b0eb-12b218b50437)

I'd jusrt change the one above to "Gasverbrauch Heizung heute" in German for consistency with the one for fuel cells and DHW. But that is for Lokalize, not here.

Reply:
This entity show the gas consumption that is used to produce electricity via a fuel cell.

Reply:
Sure, but there is no problem here, this entity reports the cubic meters of gas consumed.

And the gas  burned is used for both electricity and heat!

Reply:
You mean on a fuel cell there is no distinction between gas consumed for heating and electricity?

Reply:
Sure, how do you want to differentiate that? Depending on the state of the fuel cell the gas consumed is split in some changing percentages for electricity and waste heat which is mostly recovered for warm water and heating.
You even don't know exactly how much is going into heating and how much into hot water storage.

As I said: My whole bug report is not about the gas consumption entities, they are alll 100% correct as they report the cubic meters burned. There is no need to change those in any way.

The misleading entities are those for "heating energy" or "DHW energy" as this is the electricity consumed by the pump and electronic in a gas boiler or the electricity consumed by a heat pump.

Reply:
If you have a look at the sample data the fuel cell has
<img width="522" alt="Bildschirmfoto 2024-10-08 um 08 21 11" src="https://github.com/user-attachments/assets/67a930d0-f09b-40bb-ab83-0701c4c94662">

https://github.com/search?q=repo%3Aopenviess%2FPyViCare+%5C%22heating.gas.consumption+path%3A**%2FVitovalorPT2.json&type=code

Reply:
Those Vitovalor PT2 include a standard gas boiler alomg with the fuel cell, as you need much more heat in the winter than the fuel cell provides. Much, much more:

https://www.viessmann.de/de/produkte/brennstoffzelle/vitovalor-pt2.html

So those entities are for the gas consumed by the gas boiler. That's all correct.

And again: No need to change anything here.
Those entities report cubic meter, not energy in kWh.

Reply:
Fine, my idea was to align the naming also for the gas consumption. But we also can stick to energy.

Reply:
OK, let's look at the "name" strings of those here first:

      energy_summary_consumption_heating_currentday::name
      energy_summary_consumption_heating_lastsevendays::name
      energy_summary_consumption_heating_currentmonth::name
      energy_summary_consumption_heating_currentyear::name

I don't think it makes any sense to touch the entity keys at this point, I assume it's just fine to replace "energy" with "electricity" in their _name_ strings.

And for consistency all other "energy" entities might be changed, too.

Reply:
> All I suggested was to rename the "energy consumed" entities to "electricity consumed" so those are clearer for gas boilers etc.

I would suggest to stay with *energy* and not mix.

Reply:
I think we're completely "lost in translation" here …

We have jusrt two simple problems I reported here:

1. There are several "gas consumed" entities for "month" and "year" where it's unclear what they differ in. We did not address that part, yet.

2. _For a gas boiler_ any "Heating energy consumption" entity is very misleading as this does _not_ mean the energy contained in the gas burned but only the _tiny_ amount of electricity consumed for running the circulation pump and electronics. The actual "heating energy" is only counted as cubic meters of gas in the "Heating gas consumption" entities.

All I proposed was to make this clearer without messing around with any other entities.

I don't see any other problem that we need to fiddle around with, unless we have a very, very clear picture of what we're actually doing here.